microsoft/testsuites/kselftest/kselftest-suite.py

```diff
@@ -0,0 +1,49 @@
+from typing import Any, Dict
+
+from lisa import (
+    Environment,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    schema,
+    search_space,
+    simple_requirement,
+)
+
+from lisa.testsuite import TestResult
+from lisa.util import MissingPackagesException, SkippedException
+from microsoft.testsuites.kselftest.kselftest import Kselftest
+
+@TestSuiteMetadata(
+    area="kselftest",
+    category="community",
+    description="""
+    This test suite is used to run kselftests.
+    """,
+)
+class KselftestTestsuite(TestSuite):
+    _TIME_OUT = 12000
+
+    @TestCaseMetadata(
+        description="""
+        This test case will run kselftests tests.
+        """,
+        priority=3,
+        timeout=_TIME_OUT,
```

Comment:
12000 seconds might be a large timeout value for kselftest. What is the max test runtime that you have observed?